app/build.gradle

```diff
@@ -57,8 +57,8 @@ android {
         abortOnError false
     }
     compileOptions {
-        sourceCompatibility = '1.8'
-        targetCompatibility = '1.8'
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
Any reason why you updated these compile options in this PR?

Reply:
Because of Android 4.1